.github/workflows/reusable-industrial-ci-with-cache.yml

```diff
@@ -105,15 +105,25 @@ jobs:
         run:
           wget https://raw.githubusercontent.com/ros-controls/ros2_control_ci/master/.github/issue_template_failed_ci.md -O .github/issue_template_failed_ci.md
       - uses: JasonEtco/create-an-issue@v2
-        if: ${{ always() && steps.ici.outcome == 'failure' && github.event_name == 'schedule' }}
+        if: ${{ always() && (steps.ici.outputs.build_target_workspace == "1" || steps.ici.outputs.run_target_test || steps.ici.outputs.target_test_results == "1" ) == 'failure' && github.event_name == 'schedule' }}
```

Comment:
could you please explain this expression? it compares if the result of the 3x OR inside the parenthesis equals `'failure'`?

Reply:
whoops, the `== 'failure'` probably needs to go. I'll create a branch testing this and then finish this PR today.